lang/wasm_instance_cache.hh

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) 2022-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+
+#include "cql3/functions/function_name.hh"
+#include "utils/overloaded_functor.hh"
+#include "cql3/prepared_statements_cache.hh"
+#include "wasmtime.hh"
+#include "lang/wasm.hh"
+#include <exception>
+#include <seastar/core/metrics.hh>
```

Comment:
It's enough to use metrics_registration.

Please see if you can trim the #include list here. .hh should try to reduce their dependencies.

Reply:
It should be shorter now